angularjs-portal-home/src/main/webapp/WEB-INF/web.xml

```diff
@@ -66,24 +66,24 @@
         </param-value>
       </init-param>
     </servlet>
-    
+
     <servlet-mapping>
         <servlet-name>rest-proxy-api</servlet-name>
         <url-pattern>/api/*</url-pattern>
     </servlet-mapping>
-    
+
     <servlet>
       <servlet-name>StaticServlet</servlet-name>
       <jsp-file>/index.jsp</jsp-file>
     </servlet>
-    
+
     <filter-mapping>
        <filter-name>ExpiresFilter</filter-name>
        <url-pattern>/*</url-pattern>
        <dispatcher>REQUEST</dispatcher>
     </filter-mapping>
-    
-    
+
+
     <!-- Keep this in sync with angular-page.js -->
```

Comment:
What's `angular-page.js`, and does this changeset keep these `<servlet-mapping>`s in sync with it?